src/test/java/com/google/sps/IsValidGameNameTest.java

```diff
@@ -0,0 +1,189 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.ArrayList;
+import java.util.List;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.GameUtils;
+
+/** tests the IsValidGameName function */
+@RunWith(JUnit4.class)
+public final class IsValidGameNameTest {
+  
+  private GameUtils query;
+
+  private static String userID1 = "1";
+  private static String userID2 = "2";
+
+  // helper variable allows the use of entities in testing 
+  private final LocalServiceTestHelper helper =
+    new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+    query = new GameUtils();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  // Given a user has no games, the function should return true (any game name would be valid)
+  @Test
+  public void noGames() {
+
+    Entity userEntity2;
+    ArrayList<String> gameNames2;
```

Comment:
could you combine line 59 and 63 and 58 with 62? Is there are reason these things are not initialized on the same line?

Reply:
I one-lined those